share/empty.go

```diff
@@ -6,10 +6,10 @@ import (
 
 	"github.com/ipfs/go-blockservice"
 	"github.com/tendermint/tendermint/pkg/consts"
-
-	"github.com/celestiaorg/celestia-node/ipld"
 )
 
+//TODO: remove?
+
```

Comment:
We may potentially move this to the node's share module or into the `ipld` subpkg. At first, I saw this `Ensure..` check as application specific and thus to the node's module, but then I realized that this file is largely `ipld` related and the `ipld` also has imports to app's consts like this file. Overall, it's still unclear to me what exactly to do.